_docs/prerequisites/firewalls.md

```diff
@@ -23,34 +24,51 @@ StorageOS requires a few ports to be open:
 | 5710        | tcp       | NATS service           |
 | 5711        | tcp & udp | Gossip service         |
 
-StorageOS also uses random ports to dial-out to these ports on other StorageOS nodes. For this reason, outgoing traffic should be enabled.
+StorageOS also uses [ephemeral](https://en.wikipedia.org/wiki/Ephemeral_port) ports to dial-out to these ports on other StorageOS nodes. For this reason, outgoing traffic should be enabled.
```

Comment:
Not related to this PR, but 5701 and 5702 are internal-only and no inter-node connectivity is required.

Reply:
Same with 5704

Reply:
Might just want to mark them as "reserved"

Reply:
I'd argue explicit is better - even intra-host comms will show via netstat/tcpdump/etc. and it's the sort of detail appreciated by some.